internal/test/factory.go

```diff
@@ -38,13 +38,21 @@ func RandomBytes(size int) []byte {
 	return b
 }
 
-func GenerateBlobTxWithNamespace(namespaces []namespace.Namespace, blobSizes []int) []byte {
+func GenerateBlobTxWithNamespace(namespaces []namespace.Namespace, blobSizes []int, version uint8) []byte {
 	blobs := make([]*blob.Blob, len(blobSizes))
 	if len(namespaces) != len(blobSizes) {
 		panic("number of namespaces should match number of blob sizes")
 	}
+	var err error
```

Comment:
[nit]
why is this declared here? given that we panic if we find an error

Reply:
We need to declare it ahead of line 52 where it gets assigned